net/jool/Makefile

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright (C) 2016 Dan Luedtke <mail@danrl.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=jool
+PKG_VERSION:=3.5.1
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE_PROTO:=git
+#PKG_SOURCE_URL:=https://github.com/NICMx/Jool.git
+#PKG_SOURCE_VERSION:=TBD-after-upstream-merge
+##### v v v This is temporary. Waiting for upstream merge.
+PKG_SOURCE_URL:=https://github.com/danrl/Jool.git
+PKG_SOURCE_VERSION:=c3463caf91781c1ebb40249f236f6ae028c88f74
+##### ^ ^ ^ This is temporary. Waiting for upstream merge.
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_SOURCE_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_VERSION).tar.gz
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_SUBDIR)
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DEPENDS:=USE_UCLIBC:argp-standalone USE_MUSL:argp-standalone
```

Comment:
This line was authored by @jow- and finally made the Makefile work. Would like to consult him on this to better understand what is the best way.